codes/quantum/qubits/cubic_rbh.yml

```diff
@@ -0,0 +1,20 @@
+code_id: cubic_rbh
+
+name: 'Cubic RBH code'
+introduced: '\cite{arXiv:1805.01474}'
+
+description: |
+  Stub. (see Sec. III E of \cite{arXiv:1805.01474})
+
+protection: |
+  The symmetric energy barrier grows linearly with the lattice width. When the system is coupled locally to a thermal bath respecting the symmetry and below a critical temperature, the memory time grows expoenetially with the lattice width.
+
+relations:
+  parents:
+    - code_id: symmetry_protected
+      detail: ''
+    - code_id: qubit_stabilizer
+      detail: ''
+  cousins:
+    - code_id: surface
+      detail: 'Without the symmetry protection, the system is effectively a surface code on a face of its boundary.'
```

Comment:
Please clarify "on a face of its boundary".

Reply:
I think the name of the face depends on what will be written in the description. Maybe "the front face"? Or "the face with the toric-code boundary condition"? But it is still not defined in this stub.

Reply:
"the face with the toric-code boundary condition" sounds OK for now. As long as it's clear there's a way to do BC such that the state (or subspace?) on one face is similar to (or, modulo projecting out the bulk, equal?) to the toric code.